app/config/routing.yml

```diff
@@ -334,6 +334,14 @@ profile:
     requirements:
         id: '[a-z0-9-]+'
 
+promotional-collection:
+    path: /highlights/{id}/{slug}
```

Comment:
Unexpected :thinking:

Reply:
> Unexpected 🤔

Yes. I should have explained. I will add more detail in the associated ticket but this is the url that the external relations team have chosen.

Reply:
Have added a comment here: https://github.com/elifesciences/issues/issues/5528#issuecomment-610256138

Reply:
Causes confusion with https://github.com/elifesciences/api-raml/blob/537aff4efe86f706d604b3c368268ab5ad85fc58/src/api.raml#L1044 which I guess is not visible to the marketing people that opted for this path. Is it negotiable?

Reply:
Answer: no.

So we'll have to live with this as the only improvement would be to rename the existing `/highlights` API to resolve the confusion, which isn't warranted given the costs. Document this URL decision wherever you feel it would be read by the next person making a change.

Reply:
> Causes confusion with https://github.com/elifesciences/api-raml/blob/537aff4efe86f706d604b3c368268ab5ad85fc58/src/api.raml#L1044 which I guess is not visible to the marketing people that opted for this path. Is it negotiable?

I have had this discussion and made marketing aware of the potential confusion. The decision rested with Jen but I am open to this being discussed again. Maybe get the stakeholders in the same google meet for a few minutes?

Reply:
Looking at how `highlight` is used as a term, it is not visible in existing Journal URLs so they are allowed to choose it. Wouldn't revisit.